base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -57,12 +52,24 @@ pub struct ConsensusConstants {
     pub(in crate::consensus) emission_decay: f64,
     /// This is the emission curve tail amount
     pub(in crate::consensus) emission_tail: MicroTari,
-    /// This is the initial min difficulty for the difficulty adjustment
-    min_pow_difficulty: (Difficulty, Difficulty),
     /// The offset relative to the expected genesis coinbase value
     genesis_coinbase_value_offset: MicroTari,
     /// This is the maximum age a monero merge mined seed can be reused
     max_randomx_seed_height: u64,
+    /// This keeps track of the block split targets and which algo is accepted
+    /// Ideally this should count up to 100. If this does not you will reduce your target time.
+    algo_split: HashMap<PowAlgorithm, Algo_Time>,
+}
+
+// This is just a convenience  wrapper to put all the info into a hashmap per diff algo
+#[derive(Clone)]
+pub struct Algo_Time {
```

Comment:
nit: I think it's better not to have inconsistent abbreviations (Algo vs Algoritm)
```suggestion
pub struct PowAlgorithmConstants {
```